dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/WebContainer.java

```diff
@@ -403,7 +403,7 @@ private static Object loadListener(String lClassName) {
     // 112102 - added method below to fill the cipher to bit size table
     protected void loadCipherToBit() {
         boolean keySizeFromCipherMap =
-                        Boolean.valueOf(WebContainer.getWebContainerProperties().getProperty("com.ibm.ws.webcontainer.keysizefromciphermap", "true")).booleanValue();
+                        Boolean.parseBoolean(WebContainer.getWebContainerProperties().getProperty("com.ibm.ws.webcontainer.keysizefromciphermap", "true"));
```

Comment:
Is Boolean.parseBoolean faster than Boolean.valueOf ?  Our WCCustomProperties are using lot of valueOf.  If it is faster, we may also consider update WCCustomProperties.

Reply:
parseBoolean returns a boolean.  valueOf returns a Boolean.  So we will end up created a Boolean if we use valueOf and then call booleanValue on it to convert it to a boolean.  That is completely unnecessary.  In WCCustomProperties I believe the fields are defined as Boolean's so valueOf is the right thing if the type is Boolean.